receiptui/src/test/resources/prepaid_card_receipt_list_response.json

```diff
@@ -46,7 +46,7 @@
     {
       "journalId": "FISA_5240223",
       "type": "ADJUSTMENT",
-      "createdOn": "2019-03-31T23:55:17",
+      "createdOn": "2019-03-30T23:55:17",
```

Comment:
Why is this changed?

Reply:
@skoong due to 31st, in IST when I try to run, it moved to April 1st and I see two receipts splits to april and march, and it fails the UI. hence I changed it.

Reply:
Please change it back, once your timezone function should be able to handle this.

Reply:
Yes its done, and i changed back to 31st.